pkgs/development/tools/ocaml/merlin/default.nix

```diff
@@ -2,13 +2,15 @@
 
 buildDunePackage rec {
   pname = "merlin";
-  version = "3.3.6";
+  version = "3.3.7";
 
   minimumOCamlVersion = "4.02.1";
 
+  useDune2 = true;
```

Comment:
This change is not expected, is it? It might be worth reporting the issue upstream…

Reply:
Reported upstream here https://github.com/ocaml/merlin/issues/1157